www/src/components/layout/WithApplicationUpdate.js

```diff
@@ -1,48 +1,55 @@
-import { useCallback, useContext, useState } from 'react'
+import {
+  useCallback, useContext, useEffect, useState,
+} from 'react'
 
-import * as serviceWorker from '../../serviceWorker'
+import * as serviceWorker from '../../serviceWorkerRegistration'
 
 import { PluralConfigurationContext } from '../login/CurrentUser'
 
 const COMMIT_KEY = 'git-commit'
 
 export const DEFAULT_COMMIT = 'plural-default-commit'
 
-export const getCommit = () => localStorage.getItem(COMMIT_KEY) || DEFAULT_COMMIT
-export const setCommit = sha => localStorage.setItem(COMMIT_KEY, sha)
+export const getCommit = () => sessionStorage.getItem(COMMIT_KEY) || DEFAULT_COMMIT
```

Comment:
this could be done in a separate pr but we should actually inject the commit using an env var while building the docker container instead of storing it in local storage.  That will be a lot more reliable overall.